packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -87,6 +99,11 @@ export interface Field<T = any, V = Vector<T>> {
    * Convert a value for display
    */
   display?: DisplayProcessor;
+
+  /**
+   * Get value data links with variables interpolated
+   */
+  getDataLinksSupplier?: (config: ValueLinkConfig) => LinkModelSupplier<Field>;
```

Comment:
Alt name ideas:
* getLinksSupplier() 
* getLinksProvider()
* getLinksEvaluator)

Would be nice if we could make this more similar to the display processor: 
the display function that returns a DisplayProcessor.  
export type DisplayProcessor = (value: any) => DisplayValue;

LinkProcessor = (config: ValueLinkConfig} => Link[]

Not sure we need dual lazy eval where with a function that returns an object with a getLinks function